Library/ViewModels/AddNewCardViewModelTests.swift

```diff
@@ -12,12 +12,13 @@ internal final class AddNewCardViewModelTests: TestCase {
   private let activityIndicatorShouldShow = TestObserver<Bool, NoError>()
   private let addNewCardFailure = TestObserver<String, NoError>()
   private let addNewCardSuccess = TestObserver<String, NoError>()
+  private let cardBrandUnsupportedErrorMessageHiddenObserver = TestObserver<Bool, NoError>()
```

Comment:
Can we remove the `Observer` suffix? We've previously discussed this with @justinswart in regards to the naming of our test props